cpp/src/arrow/compute/kernels/vector_replace.cc

```diff
@@ -83,11 +87,19 @@ struct CopyArrayBitmap {
     BitUtil::SetBitTo(out_bitmap, out_offset,
                       BitUtil::GetBit(in_bitmap, in_offset + offset));
   }
+
+  void SetBitTrue(uint8_t* out_bitmap, int64_t out_offset, int64_t offset) const {
+    BitUtil::SetBitTo(out_bitmap, out_offset, true);
+  }
```

Comment:
I would suggest to change method name to `SetBitmapToTrue`

Reply:
And just for reference, BitUtil calls this `SetBit`: https://github.com/apache/arrow/blob/eb2fa2df6dedf2743047df9dd19d7ed4f4482c22/cpp/src/arrow/util/bit_util.h#L309

Reply:
solved